locales/en.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
@stuartf did you have a way in mind to sync this with crowdin? It would be preferential if the i18n keys could be part of the same project as 3akai-ux. I believe this could be done by adding a fourth location in 3akai-ux/tools/crowdin/crowdin.template that points to ep_oae's locales/en.json file.